packages/sdk/docs/README.md

```diff
@@ -0,0 +1,24 @@
+@eth-optimism/sdk / [Exports](modules.md)
```

Comment:
Is there a need to duplicate the README content from the package? Instead we can have an SDK page in community hub docs which links to docs here, same way contract API docs are linked.

Reply:
I'm not sure, this is just the auto-generated output of running `typedoc`.